src/components/CreateRules/UploadRules.vue

```diff
@@ -152,24 +161,40 @@ import FileUploadPreview from '@/components/common/FileUploadPreview.vue'
     FileUploadPreview
   }
 })
-export default class UploadRules extends Mixins(DocumentMixin) {
+export default class UploadRules extends Mixins(CommonMixin, DocumentMixin) {
+  // Refs
+  $refs!: {
+    confirmRulesChk: FormType
+  }
+
+  private INPUT_FILE_LABEL = 'Rules of Association'
```

Comment:
This is what has been done previously for "constants": https://github.com/bcgov/business-filings-ui/blob/7fd2491fb01cfc765e6284c014dec19f65121829/src/mixins/date-mixin.ts#L5

Would it make sense to do the same thing here?

Reply:
I think `private INPUT_FILE_LABEL = 'Rules of Association'` is specific enough to stay here.  But I did add `UPLOAD_FAILED_MESSAGE` and `MAX_FILE_SIZE` to `document-mixin.ts`